### DISCUSSION TITLE
How to store database and file backup separately?

### DISCUSSION THREAD

**Discussion by know-that** [spatie/laravel-backup#1535] [Q&A]
as follows:
![image](https://user-images.githubusercontent.com/31246220/173006502-299f418d-88d5-47b5-8565-caa32d9dfd0a.png)

This will cause my compressed package to be overwritten.
help me.

**Comment by erikn69** [comment]
**Duplicate of https://github.com/spatie/laravel-backup/discussions/1526**, read other discussions or issues before opening a new one

You can create two disks, `files_backups`,`db_backups`
https://github.com/spatie/laravel-backup/blob/e011c04ef4aa6ba596321675d5f6a57863d25f4d/src/Commands/BackupCommand.php#L12
```batch
backup:run --only-files --only-to-disk=files_backups
backup:run --only-db --only-to-disk=db_backups
```

Or use a command wrapper changing config with distinct `file_name_prefix` for each one
https://github.com/spatie/laravel-backup/blob/f3ef742cee592bdc1b1c14fc4b0f3afb5235a126/config/backup.php#L113

Many ways, just read documentation and use your imagination
